codegen/generate_tests.py

```diff
@@ -106,6 +113,20 @@ def main() -> None:
                 entity_snake_case=to_snake_case(entity_type.__name__),
             )
         )
+        if entity_type.__top_level__ and entity_type.__name__ not in {
+            "ProduceRequest",  # Records
+            "FetchResponse",  # Records
+            "FetchSnapshotResponse",  # Records
+            "CreateTopicsResponse",  # Should not output tagged field if its value equals to default
+            "FetchRequest",  # Should not output tagged field if its value equals to default (presumably)
```

Comment:
question(non-blocking): Does this have to do with the "ignorable" feature? I'm happy to admit I don't fully understand how it's to be interpreted and I think I concluded it doesn't apply to kio, because of the split into separate entities based on version.

Reply:
I haven't figured out yet, unfortunately